client-dynamodb/build.gradle.kts

```diff
@@ -52,6 +52,11 @@ dependencies {
       strictly("4.9.3")
     }
   }
+
+  // Required for M1 macs
+  if (org.apache.tools.ant.taskdefs.condition.Os.isArch("aarch64")) {
+    testImplementation("io.github.ganadist.sqlite4java:libsqlite4java-osx-aarch64:1.0.392")
```

Comment:
I don't think anyone else needs this? I wonder why this came up?

Reply:
Removed until I can find some definitive answer.